tests/codegen/issues/issue-74938-array-split-at.rs

```diff
@@ -0,0 +1,14 @@
+//@ compile-flags: -O
+
+#![crate_type = "lib"]
+
+const N: usize = 3;
+pub type T = u8;
+
+// CHECK-LABEL: @split_mutiple
```

Comment:
```suggestion
// CHECK-LABEL: @split_multiple
```
I'm super confused about why this test doesn't fail in CI. It does not have min-llvm-version, so it should be running :/

Reply:
Wow, looks like the llvm-17 job does not run codegen tests, filed https://github.com/rust-lang/rust/issues/126170.

Insert "you had one job" meme here.